frontends/ol-util/src/components/PageBanner.tsx

```diff
@@ -9,10 +9,22 @@ const tallBannerHeight = "252px"
 const shortMobileBannerHeight = "115px"
```

Comment:
The main change in this file is a new component `BannerPage`... Previously this file exported a bunch of components `BannerPageWrapper`, `BannerPageHeader`, etc.

It still exports those components. But most of the time we want all those components in a specific layout. So if that's what you want, now you can just use `BannerPage`.